File: SubletUC.Server/Models/Property.cs
using System.Collections.Generic;
using Microsoft.AspNetCore.Http;

public class Property
{
    public int? Id { get; set; }
    public string Address { get; set; }
    public decimal? Rent { get; set; }
    public bool? UtilitiesIncluded { get; set; }
    public decimal? AverageMonthlyUtilities { get; set; }
    public int? NumberOfBathrooms { get; set; }
    public int? NumberOfRoommates { get; set; }
    public int? Males { get; set; }
    public int? Females { get; set; }
    public int? OtherGender { get; set; }
    public string Description { get; set; }
    public string Availability { get; set; }
    public string Pets { get; set; }
    public bool? WasherDryer { get; set; }
    public bool? DrivewayParking { get; set; }
    public bool? StreetParking { get; set; }
    public List<IFormFile> Pictures { get; set; }

    public Property()
    {
        Pictures = new List<IFormFile>();
    }
}

File: SubletUC.Client/src/app/models/property.ts
export class Property {
    id?: number;
    address?: string;
    rent?: number;
    utilities_included?: boolean;
    average_monthly_utilities?: number;
    number_of_bathrooms?: number;
    number_of_roommates?: number;
    males?: number;
    females?: number;
    other_gender?: number;
    description?: string;
    availability?: string;
    pets?: string;
    washer_dryer?: boolean;
    driveway_parking?: boolean;
    street_parking?: boolean;
    pictures?: File[]; // Update this line to an array of Files

  constructor() {
    this.pictures = [];
  }
}